### PR TITLE
Task-2: Adding Character's Classes, Attributes and Health/Mana Bar

### PR DESCRIPTION
**Pull request by gabrielcicero45**
`refactor:`  Organizing components architecture.
image

### CODE REVIEW

**Review by bioneoficial — APPROVED** [review]

File: src/components/HealthAndMana/HealthAndMana.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    <div className="sheet__item">
+      Health: {health>maxHealth ? maxHealth:health}/{maxHealth}
+      <div class="bar">
+        <div class="health-bar" style={{width: `${health/maxHealth * 100}%`}}></div>
+      </div>
+    </div>
+    <div className="sheet__item">
+      Mana: {mana>maxMana ? maxMana:mana}/{maxMana}
+      <div class="bar">
+        <div class="mana-bar" style={{width: `${mana/maxMana * 100}%`}}></div>
```
Comment (julialbq) [inline src/components/HealthAndMana/HealthAndMana.js:20]: I really liked the idea of the mana bar and the health bar!

File: src/components/RpgClass/RpgClass.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+import paladin from '../../assets/icons/paladin.jpeg'
+import ranger from '../../assets/icons/ranger.jpeg'
+import rogue from '../../assets/icons/rogue.jpeg'
+import sorcerer from '../../assets/icons/sorcerer.jpeg'
+import warlock from '../../assets/icons/warlock.jpeg'
+import wizard from '../../assets/icons/wizard.jpeg'
+
+export default function RpgClass({rpgClass}) {
+  const findIcon = (rpgClass)=>{
+    console.log(rpgClass)
```
Comment (julialbq) [inline src/components/RpgClass/RpgClass.js:19]: remove the console.log

**Review by julialbq — APPROVED** [review]

File: src/components/RpgClass/RpgClass.js (hunk 2 above)
Comment (bioneoficial) [inline src/components/RpgClass/RpgClass.js:19]: You still need this console.log in your application? If not, please remove!

File: src/components/Sheet/Sheet.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import Attributes from '../Attributes/Attributes'
+import RpgClass from '../RpgClass/RpgClass'
+import HealthAndMana from '../HealthAndMana/HealthAndMana'
+import LevelAndXp from '../LevelAndXp/LevelAndXp'
+import Name from '../Name/Name'
+import './Sheet.css'
+export default function Sheet({player}) {
+  const {name,health,mana,xp,attributes,rpgClass} =player
```
Comment (bioneoficial) [inline src/components/Sheet/Sheet.js:9]: Please add an 'space' between '=' and 'player'

File: src/components/Sheet/Sheet.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  }
+
+  return (
+    <>
+    <div className='sheet__container'>
+      <div className="sheet__column">
+        <Name playerName={name}/>
+      </div>
+      <div className="sheet__column">
+        <HealthAndMana health={health} mana={mana} level={calculateLevel(xp)} />
```
Comment (bioneoficial) [inline src/components/Sheet/Sheet.js:34]: Some components you add an space between the last prop and the '/>'.
Please choose a pattern, to be honest idk the best practice, let ask the mentors, with or without space.

**Review by bioneoficial — COMMENTED** [review]
good work

**Pull request merged**